_posts/blog/2022-04-26-march-2022-community-feedback.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+categories: 
+  - blog
+title: "March 2022 Community Feedback Session"
+description: "On March 15, 2022, Chi Hack Night held a hybrid community feedback session
+to assess how our community is feeling about the current phase of the pandemic and what people
+are looking for from Chi Hack Night in this moment."
+date: 2022-04-26
+image: /images/blog/2022-04-26-march-2022-community-feedback/20220315_community_feedback_miro_board.png
```

Comment:
files are now `jpg` so we need to change this from `png`

Reply:
Thanks 🙏